[DOCS] Fixes more small issues noticed during onboarding (#10598)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/expectations/manage_expectations.md

| Data Integrity     | `expect_column_pair_values_to_be_equal`                   | Expect the values in column A to be the same as column B.                                                                              |
| Data Integrity     | `expect_multicolumn_sum_to_equal`                         | Expect that the sum of row values in a specified column list is the same for each row, and equal to a specified sum total.             |
| Distribution       | `expect_column_pair_values_A_to_be_greater_than_B`        | Expect the values in column A to be greater than column B.                                                                             |

> | Distribution       | `expect_column_values_to_be_between`                      | Expect the column entries to be between a minimum value and a maximum value.                                               |

Author: **note for reviewers**: I removed "(inclusive)" because the user can make these expectations either inclusive or exclusive depending on how they set "Strict Min" / "Strict Max" in the UI

| Missingness        | `expect_column_values_to_not_be_null`                     | Expect the column values to not be null.                                                                                               |
| Numerical Data     | `expect_column_max_to_be_between`                         | Expect the column maximum to be between a minimum and a maximum value.                                                                 |
| Numerical Data     | `expect_column_mean_to_be_between`                        | Expect the column mean to be between a minimum and a maximum value.                                                        |
| Numerical Data     | `expect_column_median_to_be_between`                      | Expect the column median to be between a minimum and a maximum value.                                                                  |
| Numerical Data     | `expect_column_min_to_be_between`                         | Expect the column minimum to be between a minimum value and a maximum value.                                                           |

> | Pattern matching   | `expect_column_value_length_to_equal`                     | Expect the column entries to be strings with length equal to the provided value.                           |

Author: **note for reviewer** This definition was wrong, it looked like copy/pasta to me.

> | Pattern matching   | `expect_column_value_length_to_equal`                     | Expect the column entries to be strings with length between a minimum value and a maximum value.                           |

Author: **note for reviewers** this expectation description was wrong, it looks like copy/pasta to me.

> | Pattern matching   | `expect_column_value_length_to_equal`                     | Expect the column entries to be strings with length between a minimum value and a maximum value.                           |

Author: ```suggestion
| Pattern matching   | `expect_column_value_length_to_equal`                     | Expect the column entries to be strings with length equal to the provided value.                           |
```
